bubblewrap.c

```diff
@@ -1378,6 +1380,17 @@ parse_args_recurse (int          *argcp,
           argv += 1;
           argc -= 1;
         }
+      /* should be the last nominal parameter */
+      else if (strcmp (arg, "--exec-filename") == 0)
+        {
+          if (argc < 2)
+            die ("--exec-filename takes an argument");
+
+          opt_exec_filename = argv[1];
+	  argv++;
```

Comment:
I don't know what happened, but these lines ended up being unindented..

Reply:
Yes, please fix the indentation.

Also, the other options mostly seem to use `argc -= 1; argv += 1;` (although the current code is not entirely consistent) so it's probably better to be consistent with that form.